findspark.py

```diff
@@ -35,7 +37,79 @@ def find():
     return spark_home
 
 
-def init(spark_home=None):
+def change_rc(spark_home, spark_python, py4j):
+    """Persists changes to enviornment by changing shell config.
+
+    Adds lines to .bashrc to set enviornment variables including
+    the adding of dependencies to the system path. Currently only 
+    works for Bash and (t)csh.
+
+    Parameters
+    ----------
+    spark_home : str
+        Path to Spark installation.
+    spark_python : str
+        Path to python subdirectory of Spark installation.
+    py4j : str
+        Path to py4j library.
+    """
+
+    bashrc_location = os.path.expanduser("~/.bashrc")
+
+    with open(bashrc_location, 'a') as bashrc:
+        bashrc.write("\n# Added by findspark\n")
+        bashrc.write("export SPARK_HOME=" + spark_home + "\n")
+        bashrc.write("export PYTHONPATH=" + spark_python + ":" + 
+                     py4j + ":$PYTHONPATH\n\n")
+    
+    cshrc_location = os.path.expanduser("~/.cshrc")
+    
+    with open(cshrc_location, 'a') as cshrc:
+        cshrc.write("\n# Added by findspark\n")
+        cshrc.write("setenv SPARK_HOME " + spark_home + "\n")
+        cshrc.write("setenv PYTHONPATH \"" + spark_python + ":" +
+                    py4j + ":\"$PYTHONPATH")
+ 
+
+def edit_ipython_profile(spark_home, spark_python, py4j, name):
```

Comment:
The idea is: wait for people to need functionality, because it's easier to add things to APIs than remove them. Also, with the notebook and kernelspecs, it's becoming a bit less common for people to use IPython profiles other than default.